src/main/java/seedu/address/model/person/Person.java

```diff
@@ -34,6 +35,21 @@ public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tag
         this.email = email;
         this.address = address;
         this.tags.addAll(tags);
+        this.nric = new Nric("S1234567Z"); //Placeholder NRIC
```

Comment:
Placing a placeholder value might not be a good idea without adding / modifying the existing tests. 

Also, perhaps we can use an `Optional<String> NRIC` to represent the NRIC?

Reply:
Ah but I was thinking cause NRIC should be a compulsory field due to the non-null and valid checks, and the placeholder is for enforcement of previous tests. For future implementations, we should be able to remove this original constructor and use the one that includes the NRIC.

Reply:
Ahh I see. Okay!